app/src/main/res/layout/interation_feedback_card_item.xml

```diff
@@ -0,0 +1,25 @@
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
```

Comment:
For file name, should this be ``interaction_feedback_card_item``?

Reply:
How shall I name it then, it relates to the learners interaction that should be shown on the right side.